### DISCUSSION TITLE
Stellarium Web Home Page

### DISCUSSION THREAD

**Discussion by xytsrm** [Stellarium/stellarium#3635] [General]
The background color of dark-gray makes it almost impossible to read any black text. (e.g. Black text description of download versions at the top of the page).

X.

**Comment by gzotti** [comment]
Assuming you mean Stellarium's home page, not the one of Stellarium Web (which has no download icons). 
The upper area is brightened up to mid gray, to allow reading the text.  Is your environment too bright?

**Reply by xytsrm** [reply]
The shade of gray on three browsers is dark-gray; Black text on gray is never good. Most of the text on that page is white; all text should be white.
